jib-core/src/test/java/com/google/cloud/tools/jib/configuration/BuildConfigurationTest.java

```diff
@@ -250,6 +253,17 @@ public void testBuilder_nullValues() {
       Assert.assertNull(ex.getMessage());
     }
 
+    // Labels element should not be null.
+    try {
+      Map<String, String> badLabels = new HashMap<>();
+      badLabels.put("label-key", null);
+      BuildConfiguration.builder(Mockito.mock(BuildLogger.class))
+          .setContainerConfiguration(ContainerConfiguration.builder().setLabels(badLabels).build());
```

Comment:
The try block should only be focused on the method being tested. I actually think `BuildConfiguration.builder()` doesn't play a role. Maybe

```java
Map<String, String> badLabels = ...
badLabels.put(...)
Builder builder = ContainerConfiguration.builder();
try {
  builder.setLabels(badLabels);
  Assert.fail(...);
} catch (...) {
```

Reply:
Yeah, makes sense.

Reply:
hrmm... looks like we might need to cleanup this whole test class :\